client/i18n/locales/english/intro.json

```
@@ -3522,6 +3522,12 @@
           "In this lab, you will practice using higher order functions to find the symmetric difference between two arrays."
         ]
       },
+      "lab-matching-object-filter": {
+        "title": "Implement a Matching Object Filter",
+        "intro": [
+          "In this lab, you will create a function that looks through an array of objects and returns an array of all objects that have matching property and value pairs."
+          ]
+      },
```

The linting error might be due to the whitespace character:

```suggestion
      "lab-matching-object-filter": {
        "title": "Implement a Matching Object Filter",
        "intro": [
          "In this lab, you will create a function that looks through an array of objects and returns an array of all objects that have matching property and value pairs."
        ]
      },
```